services/src/handlers/ebv.rs

```diff
@@ -51,8 +51,9 @@ where
             web::resource("/dataset/{id}/subdatasets")
                 .route(web::get().to(get_ebv_subdatasets::<C>)),
         )
-        .service(web::resource("/create_overviews").route(web::post().to(create_overviews::<C>)))
-        .service(web::resource("/create_overview").route(web::post().to(create_overview::<C>)));
+        .service(web::resource("/createOverviews").route(web::post().to(create_overviews::<C>)))
+        .service(web::resource("/createOverview").route(web::post().to(create_overview::<C>)))
+        .service(web::resource("/removeOverview").route(web::post().to(remove_overview::<C>)));
```

Comment:
why not use DELETE http method?

Reply:
[bd83581](https://github.com/geo-engine/geoengine/pull/592/commits/bd835810a808f93fa3666b9d715a6b03f9c3ea6f)